src/i18n/messages.json

```diff
@@ -74,6 +74,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
+        "name": "Angela Busato",
+        "github": "AngyDev",
+        "github_url": "https://github.com/AngyDev",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
+        "twitter_url": "https://twitter.com/angela_busato",
+        "image": "/img/_dev/Agny.jpg",
```

Comment:
Also in the name of the img there is the same typo

Reply:
done